Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/TransformTool.java

```diff
@@ -257,7 +257,7 @@ private void onApplyResizeClicked(int resizePercentage) {
 
 		if (newWidth == 0 || newHeight == 0) {
 			zeroSizeBitmap = true;
-			contextCallback.showNotification(R.string.resize_cannot_resize_to_this_size, ContextCallback.NotificationDuration.LONG);
+			contextCallback.showNotificationWithDuration(R.string.resize_cannot_resize_to_this_size, ContextCallback.NotificationDuration.LONG);
```

Comment:
I added this because, in the PR in which I added lint checks for Jenkins, it was giving a warning because of the same name for two methods 🤷🏻‍♂️, so I added it here too.